.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "monthly"
+    open-pull-requests-limit: 1
```

Comment:
```suggestion
    open-pull-requests-limit: 99
```

Reply:
I just copied this from the setup in FHIRClient, I guess we should change it there as well?